scripts/detect_timeout.py

```diff
@@ -1,7 +1,45 @@
 import argparse
 import re
+from nltk import word_tokenize
```

Comment:
I am intentionally using only packages that comes with base installation. I would avoid heaving dependency for such small job.

Reply:
Eh looks like this import was not even used